src/main/java/org/zalando/planb/provider/CassandraConfig.java

```diff
@@ -19,6 +20,7 @@ Session initializeCassandra() {
         final Cluster.Builder builder = Cluster.builder();
 
         builder.addContactPoints(cassandra.getContactPoints().split(","));
+        builder.withAddressTranslator(new EC2MultiRegionAddressTranslator());
```

Comment:
Hm, not sure about a bool flag. For example, something similar might be needed for GCE or we might want to plug a custom implementation.